assets/src/scripts/outputs-viewer/index.jsx

```diff
@@ -1,7 +1,18 @@
+import * as Sentry from "@sentry/react";
 import React from "react";
 import ReactDOM from "react-dom";
 import App from "./App";
 
+if (
+  process.env.NODE_ENV === "production" &&
+  document.location.hostname === "jobs.opensafely.org"
+) {
+  Sentry.init({
+    dsn: "https://adf34e0d9a5445fea760b43cf10ad45b@o173701.ingest.sentry.io/5881265",
```

Comment:
Can we build this from an env var?  We have that available in production as `SENTRY_DSN`, and ideally we'd not expose per-env config in the code.

Reply:
Yes, I've added it as a Vite env var which needs to be available when the assets are built. I will get it added to the GitHub Actions.